packages/contracts/waffle.js

```diff
@@ -11,10 +11,9 @@ var waffleConfig = {
 };
 
 var selectSolc = () => {
-  if (process.env.CI || process.env.NATIVE_SOLC == "true") {
+  if (1) {
```

Comment:
hm

Reply:
shouldn't change this, because Netlify doesn't have `solc` binary natively installed, which will fail all checks.

Reply:
this is just for me, since my .env file has never worked...